torch/_inductor/scheduler.py

```diff
@@ -1223,6 +1223,10 @@ def __init__(self, nodes):
         # for debug attribution
         self.origin_to_index = {}
 
+        # ctypes limited 1024 args, and the worst case each time fused 2x args.
+        # let's limited args number to 500
+        self.MAX_FUSED_KERNEL_ARGS_NUM = 500
```

Comment:
Make this constant a global variable instead?

Reply:
Make it as class-level variable.